{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/templates/account/logout.html

```diff
@@ -5,18 +5,28 @@
 {% block head_title %}{% trans "Sign Out" %}{% endblock %}
 
 {% block inner %}
+
+{% if ACCOUNT_HIDE_INTERMEDIARY_LOGOUT %}
+  {# Hide form - prevents user confusion by seeing an auto-submitting form #}
+  <script>document.getElementsByClassName('container')[0].style.display = 'none';</script>
+{% endif %}
+
 <h1>{% trans "Sign Out" %}</h1>
```

Comment:
If you indeed use a `display:none` `div`, this can be left out of the `div` and becomes something like

```
<h1>{% trans "Signing Out" %}</h1>
```
so that users can see a `Sinning Out` message instead of a blank page.